src/amici.cpp

```diff
@@ -1623,6 +1997,55 @@ int workBackwardProblem(UserData *udata, TempData *tdata, ReturnData *rdata, con
     return AMICI_SUCCESS;
 }
 
+int workSteadyStateProblem(UserData *udata, TempData *tdata, ReturnData *rdata, void *ami_mem, int it) {
+    /*
+     * tries to determine the steady state of the ODE system by a Newton solver
+     uses forward intergration, if the Newton solver fails
+     *
+     * @param[in] udata pointer to the user data struct @type UserData
+     * @param[in] tdata pointer to the temporary data struct @type UserData
+     * @param[out] tdata pointer to the temporary data struct @type TempData
+     * @param[out] rdata pointer to the return data struct @type ReturnData
+     */
+    
+    int status = (int) *rdata->status;
+    double run_time;
+    clock_t starttime;
```

Comment:
now it is used again. Actually, starttime should be a clock_t type, tun_time a double. The way it was before (with starttime superfluous) was actually incorrect, since run_time was used as double and clock_t. For some reason, the compiler didn't worry. But since it is improper, I changed it back.